pkg/cli/interactive_tests/test_secure.tcl

```diff
@@ -2,7 +2,7 @@
 
 source [file join [file dirname $argv0] common.tcl]
 
-set certs_dir "certs"
+set certs_dir "/certs"
```

Comment:
this did work, but we're using the absolute path in other places, I had it switched when testing manually a few PRs ago, this is just resetting it.